src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Bugzilla/Bugzilla25662.cs

```diff
@@ -12,14 +13,23 @@ public Bugzilla25662(TestDevice testDevice) : base(testDevice)
 
     public override string Issue => "Setting IsEnabled does not disable SwitchCell";
 
-	// [Test]
-    // [Category(UITestCategories.Cells)]
-	// [FailsOnIOS]
-	// [FailsOnWindows]
-	// public void Bugzilla25662Test()
-	// {
-	// 	App.WaitForElement("One");
-	// 	App.Tap("One");
-	// 	App.WaitForNoElement("FAIL");
-	// }
+	[Test]
+	[Category(UITestCategories.Cells)]
+	[FailsOnIOS]
+	[FailsOnMac]
+	[FailsOnWindows]
+	public void Bugzilla25662Test()
+	{
+		if (App is not AppiumApp app2 || app2 is null || app2.Driver is null)
+		{
+			throw new InvalidOperationException("Cannot run test. Missing driver to run quick tap actions.");
+		}
+
+		App.WaitForNoElement("One");
+
+		var item = app2.Driver.FindElement(OpenQA.Selenium.By.XPath("//*[@text='" + "One" + "']"));
```

Comment:
We can simplify this after merging https://github.com/dotnet/maui/pull/25177
This PR is in Draft status until be able to refactor this cases.

Reply:
That PR is merged! Should we update this first?